Cargo.toml

```diff
@@ -44,6 +44,7 @@ thiserror = { version = "2", default-features = false }
 # metrics
 metrics = { version = "0.24.1" }
 metrics-exporter-prometheus = { version = "0.16.2", default-features = false, features = ["http-listener"] }
+metrics-process = { version = "2.4.0" }
```

Comment:
Nit: just do `"2.4.0"` instead?

Reply:
Updated, as well for `metrics` and `prometheus-parse`.